### ISSUE TITLE
Can I get a thread?

### ISSUE THREAD

**Issue by scripting** [scripting/blue.feedland#2]
So now that I have a friend who knows how to make feeds out of content from Bluesky -- 

How about a thread? If I give you an id of a post can you give me a hierarchic representation of the thread?

Ideally it would be OPML which I can open directly in [Drummer](https://drummer.land/). And the [outliner](https://github.com/scripting/concord) in Drummer is open source and can read OPML too.

**Comment by johnspurlock** [comment]
Yes I think I could do that.  Could you provide an example opml output for a bluesky thread?

Threads can be massive, so it would be nice if there was a way to page the opml in some way, or point via url to another opml to dig deeper into a subsection of the reply graph.

**Comment by scripting** [comment]
@johnspurlock -- thanks for the feedback. 

i have other much more pressing questions, still getting them organized. 

i'll have a new demo later today, with an outline...
